files-csharp/CopyAllContentInDirectory/CopyAllContentInDirectoryTests/CopyAllContentInDirectoryTests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+    <PackageReference Include="Moq" Version="4.17.2" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.7" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.7" />
+    <PackageReference Include="coverlet.collector" Version="3.1.0" />
```

Comment:
Is coverlet.collector needed? System.IO.Abstractions not needed. Moq is not needed. I understand that you've been investigating how to mock the file system and that these are leftovers, but please clean the projects when you finish.

Reply:
removed